draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -901,11 +901,10 @@ Notation: x_i is the i^th element of the vector representation of x.
 Steps:
 1. sign = 0
 2. for i in (1, ..., m):
-3.   if x_i > (p - 1) / 2                  # x_i is negative
-4.     sign = CMOV(sign, -1, sign == 0)
-5.   elif x_i > 0                          # x_i is positive (and nonzero)
-6.     sign = CMOV(sign, 1, sign == 0)
-7. return CMOV(sign, 1, sign == 0)         # regard x = 0 as positive
+3.   sign_i = CMOV(1, -1, x_i > (p - 1) / 2)
```

Comment:
Should we add a matching Python/Sage implementation? (Not in this PR -- we can do that in a separate one.)

Reply:
That makes sense. The version I added to common.sage for this PR gives the same answer but probably ends up obfuscated for trying to be too clever.

I added #108 for this.